client/js/screens/Campaigns/styles.js

```diff
@@ -1,3 +1,88 @@
 import {StyleSheet} from 'react-native';
-const styles = StyleSheet.create({});
+const styles = StyleSheet.create({
+  xxxIcon: {
+    resizeMode: 'contain',
+    height: 30,
+    alignSelf: 'flex-end',
+  },
+  categoryImage: {
+    height: 60,
+    width: 100,
+    resizeMode: 'contain',
+    // alignSelf: 'flex-start',
+  },
+  campaignsHeader: {
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingTop: 10,
+    marginBottom: 10,
+    marginRight: 60,
+    marginLeft: 20,
+  },
+  categoryVideo: {
+    width: 130,
+    height: 80,
+    resizeMode: 'contain',
+    marginLeft: 15,
+  },
+  videoSection: {
+    flex: 1,
+    flexDirection: 'row',
+    height: 100,
+    marginBottom: 0,
+  },
+
+  border: {
+    marginVertical: 10,
+    borderBottomColor: 'black',
+    borderBottomWidth: 1,
+  },
+  campaignsView: {
+    flex: 1,
+    backgroundColor: 'yellow',
+  },
+  campaignSingle: {
+    alignSelf: 'center',
+  },
+  campaignSingleMainImage: {
+    width: 150,
+    height: 150,
+    // borderRadius: 15,
+    borderTopLeftRadius: 15,
+    borderTopRightRadius: 15,
+    // margin: 20,
```

Comment:
Please use image resize mode for your images, %s for height and width and remove comments.

Reply:
I was able to remove the width fixed height but no matter what I tried the height of 150 needs to be set. I tried using resize mode.